File: Data Structures & Algorithms/maximum-difference-between-even-and-odd-frequency-i/submission-4.java
class Solution {
    public int maxDifference(String s) {
        int[] fr = new int[26];
        char c;
        for(int i = 0; i < s.length(); i++)
        {
            c = s.charAt(i);
            fr[c - 'a']+=1;
        }
        int od = Integer.MIN_VALUE,ev = Integer.MAX_VALUE;
        for(int i : fr)
        {
            if(i == 0) continue;
            if(i % 2 == 0) ev = Math.min(ev,i);
            else if(i % 2 != 0) od = Math.max(od,i);
        }
        return od - ev;
    }
}

File: Data Structures & Algorithms/maximum-number-of-balloons/submission-4.cpp
class Solution {
public:
    int maxNumberOfBalloons(string text) {
        int ans[26] = {0};
        for(char i : text) ans[i - 'a']+=1;
        return min({ans['a' - 'a'],ans['b' - 'a'],ans['n' - 'a'],ans['l' - 'a']/2,ans['o' - 'a']/2});
    }
};